File: ts/filtering-content/GetItemsByItsLinkedItem.ts
// DocSection: filtering_get_items_its_linked_item
deliveryClient.items<ContentItem>()
  // Gets items whose linked item has codename as one of specified values
  .anyFilter(“elements.authors”, [“john_snow”], ["johnny_bravo"])

deliveryClient.items<ContentItem>()
  // Gets items whose linked item has codename of certain value
  .containsFilter(“elements.authors”, [“joe_down”])
// EndDocSection

File: ruby/filtering-content/GetItemsByItsLinkedItem.rb
# DocSection: filtering_get_items_by_linked_item
# Gets items attributed to Jane.
delivery_client.items('elements.author'.contains('jane_doe'))

# Gets items attributed to at least Jane, John, or both.
delivery_client.items('elements.author'.any('jane_doe', 'john_wick'))
# EndDocSection